server/games/game.py

```diff
@@ -260,7 +263,7 @@ def _process_pending_army_stats(self):
     def _process_army_stats_for_player(self, player):
         try:
             if (
-                self._army_stats is None
+                len(self._army_stats_list) == 0
```

Comment:
`if self._army_stats_list` ;-)

Reply:
I did that, and then only noticed that it actually should be `if not self._army_stats_list or ...` when the tests failed. That promptly reminded me of why I find the `len(stuff) == 0` more readable.